net/crowdsec/Makefile

```diff
@@ -0,0 +1,189 @@
+#
+# Copyright (C) 2021 Gerald Kerma
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=crowdsec
+PKG_VERSION:=1.1.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/crowdsecurity/crowdsec
+PKG_SOURCE_VERSION:=73e0bbaf93070f4a640eb5a22212b5dcf26699de
+PKG_SOURCE_DATE:=20210707
+PKG_MIRROR_HASH:=940e6e122097468ad99fcee2db569afda94fc2a65837c74b0ee71bbb5c64518c
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Gerald Kerma <gandalf@gk2.net>
+
+PKG_BUILD_DEPENDS:=golang/host jq/host
```

Comment:
It appears the only place where jq is used during build is to [get the codename](https://github.com/crowdsecurity/crowdsec/blob/v1.1.1/Makefile#L33), but since this is now hard-coded below, I don't think this is dependency is necessary anymore.

(jq is also used in [wizard.sh](https://github.com/crowdsecurity/crowdsec/blob/v1.1.1/wizard.sh#L474) though I believe this script is only suppose to be used when building from source.)

Reply:
wizard.sh may be used by user, but it is not integrated for now
So jq dependency may be removed
jq/host is no more necessary as long as the default Makefile is no more used
removed
thanks

Reply:
wizard.sh may be used by user, but it is not integrated for now
So jq dependency may be removed
jq/host is no more necessary as long as the default Makefile is no more used
removed
thanks